astropy/coordinates/builtin_frames/lsr.py

```diff
@@ -37,25 +37,20 @@
 
 @format_doc(base_doc, components=doc_components_radec, footer=doc_footer_lsr)
 class LSR(BaseRADecFrame):
-    r"""A coordinate or frame in the Local Standard of Rest (LSR).
-
-    This coordinate frame is axis-aligned and co-spatial with
```

Comment:
I think some of this conceptual explanation is quite useful to include! The LSR is such a fuzzy concept, it's good to communicate how we are thinking about it.